dcc-portal-ui/app/scripts/ui/js/directives.js

```diff
@@ -366,9 +366,9 @@ angular.module('icgc.ui.popover', [])
 angular.module('icgc.ui.copyPaste', [])
   .provider('copyPaste', function () {
     var _provider = this,
+        //_zeroClipPath = require('../assets/ZeroClipboard.swf'), <- for local development only
         _zeroClipPath = '//cdnjs.cloudflare.com/ajax/libs/zeroclipboard/2.2.0/ZeroClipboard.swf',
```

Comment:
I'm unfortunately unfamiliar with the circumstances under which it was reinstated